### PR TITLE
Feature/game date

### PR DESCRIPTION
**Pull request by mrawji**
When looking at my favorite team's past games, I found it hard to make sure i was looking at the correct games.

This feature adds dates to that page, and leaves the other pages alone.
It also adds a "Starting at" prefix for some of the games. See the attached screenshots

My Team Recent Game - Orignal:
My Team Recent Game - Orignal

My Team Recent Game - With the Change:
My Team Recent Game - With the Change

Showing the Starting At:
Showing the Starting At

### CODE REVIEW

File: resources/lib/nhl_tv.py [hunk 1]
```diff
@@ -73,22 +72,29 @@ def create_game_listitem(game, game_day):
...
         game_time = game['gameDate']
         game_time = string_to_date(game_time, "%Y-%m-%dT%H:%M:%SZ")
         game_time = utc_to_local(game_time)
-
+        game_date = game_time.strftime("%Y-%m-%d")
         if TIME_FORMAT == '0':
             game_time = game_time.strftime('%I:%M %p').lstrip('0')
         else:
             game_time = game_time.strftime('%H:%M')
+        game_line_header = '%s %s' % (LOCAL_STRING(30023), game_time)
```
Comment (eracknaphobia) [inline resources/lib/nhl_tv.py:85]: I think displaying the start time should be enough here.

File: resources/language/resource.language.en_gb/strings.po [hunk 2]
```diff
@@ -36,6 +36,9 @@ msgctxt "#30022"
 msgid "Recap"
 msgstr ""
 
+msgctxt "#30023"
+msgid "Starting at"
```
Comment (eracknaphobia) [inline resources/language/resource.language.en_gb/strings.po:40]: I think having the start time in the label makes it apparent the game has not started.

**Review by eracknaphobia — CHANGES_REQUESTED** [review]
I really like the date in the recent games list, nice work! I just would like to slim down the start time. Thanks for the PR!

**Comment by mrawji** [comment]
Agreed. Will make another tonight or tomorrow

**Comment by mrawji** [comment]
Should be ok now (this the first time I've done a PR with a rebase. LMK if I messed it up).
Marc

**Comment by eracknaphobia** [comment]
Looks good, thanks!

**Pull request merged**